Sample/src/androidTest/java/com/shopify/testify/sample/ScreenshotRuleExampleTests.kt

```diff
@@ -216,6 +216,9 @@ class ScreenshotRuleExampleTests {
     @Test
     fun setExactness() {
         rule
+            .setScreenshotViewProvider {
+                it.findViewById(R.id.info_card)
+            }
             .setExactness(0.9f)
```

Comment:
We can maybe set this higher now? 0.9 seems quite low

Reply:
You're right. It can be higher. I'll update it